pkg/controllers/node/termination/terminator/suite_test.go

```diff
@@ -89,6 +89,11 @@ var _ = Describe("Eviction/Queue", func() {
 		})
 	})
 
+	AfterEach(func() {
+		// Reset the metrics collectors
+		terminator.PodEvictedCounter.Reset()
+	})
```

Comment:
nit: I think it's just as equivalent for testing purposes to put this in the BeforeEach() block, and it's where everything else like this is defined.

Reply:
I think we generally put cleanup in the AfterEach block. Resetting the counter is probably something that should be done at the end of the test since the value will be changed during the test